README.md

```diff
@@ -1,129 +1,241 @@
-# Hello-Microverse-Project
-First practice 
-logo
-Microverse README Template
-📗 Table of Contents
-📖 About the Project
-🛠 Built With
-Tech Stack
-Key Features
-🚀 Live Demo
-💻 Getting Started
-Setup
-Prerequisites
-Install
-Usage
-Run tests
-Deployment
-👥 Authors
-🔭 Future Features
-🤝 Contributing
-⭐️ Show your support
-🙏 Acknowledgements
-❓ FAQ
-📝 License
-📖 HELLO MICROVERSE PROJECT
-Describe your project in 1 or 2 sentences.
-
-HELLO MICROVERSE PROJECT is the first project I work on respecting the rule of a professional repo
-
-🛠 Built With
-
-
-Client : Html & CSS
-
-Key Features:
-    1 This application display "Hello Microvese"
-(back to top)
-
-🚀 Live Demo
-https://github.com/Grandi0z/Hello-Microverse-Project/tree/3efd29b3a68c4459e9a77708191b49c72aa76c78
-(back to top)
-
-💻 Getting Started
-This project will be useful if you want to work as a professional
-
-To get a local copy up and running, Visit our website
-
-Prerequisites
+
+<a name="readme-top"></a>
+
+
+
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b># Hello-Microverse-Project</b></h3>
+
+</div>
+
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+
+
+# 📖 [HELLO MICROVERSE PROJECT] <a name="about-project"></a>
+
+
+**[HELLO MICROVERSE PROJECT]** is the first project I work on respecting the rule of a professional repo
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> Describe the tech stack and include only the relevant sections that apply to your project.
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://w3.org/">HTML & CSS</a></li>
+  </ul>
+</details>
+
+
+
+
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- **[1 This application displays "Hello Microvese"]**
+- **[2 This application has a title]**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
+
+- [Live Demo Link](https://github.com/Grandi0z/Hello-Microverse-Project/tree/3efd29b3a68c4459e9a77708191b49c72aa76c78)
+
```

Comment:
***[OPTIONAL]*** `Live demo link` means the live version of your website that can be accessed from anywhere in the world. You have added the wrong link. 
You can consider adding the correct link as it is professional, and the viewer can easily find your website's online version from Readme. Here is a [FreecodeCamp Link](https://www.freecodecamp.org/news/publish-your-website-netlify-github/) for reference.